application/controllers/TicketController.php

```diff
@@ -111,30 +111,39 @@ public function changeextrasAction ()
             return $this->_helper->redirector('list', 'lan');
         if (! $this->_ticketIDValidator->isValid($form->getValue('ticketId')))
             return $this->_helper->redirector('list', 'lan');
-        
+
         $ticketId = (int) $form->getValue('ticketId');
-        
+
         $ticket = new Application_Model_Ticket();
         $this->_mapperTickets->find($ticketId, $ticket);
-        
+
         if ($this->_currentUser->getId() != $ticket->getUserId())
             return $this->_helper->redirector('list', 'lan');
-        
+
         $extras = array();
+        $hasHelpingStatusChanged = false;
         foreach ($form->getValues() as $key => $value) {
-            if ($key != "ticketId" && $key != "submit") {
+            if ($key != "ticketId" && $key != "submit" && $key != "helping") {
                 if ($value == "1") {
                     $value = true;
                     $extras[$key] = $value;
                 } elseif ($value == "0") {
                     $value = false;
                     $extras[$key] = $value;
                 }
+            } elseif ($key == "helping") {
+                $hasHelpingStatusChanged = $ticket->getHelping() != $value;
+                $ticket->setHelping($value == "1");
             }
         }
```

Comment:
Nope, one of the keys is called "helping" and we need to check that separately so we can set a true/false flag